src/poolMetas.ts

```diff
@@ -162,12 +195,27 @@ export let poolMetas: PoolMeta[] = [
     nftFeed: '0x677b85998f15921982b1548763f01ac6c265b8eb',
     assessor: '0x8b80927fca02566c29728c4a620c161f63116953',
     seniorTranche: '0x88ad5b21a01d838b15619f36f88b618410797b95',
+    juniorTranche: null,
+    drop: null,
+    tin: null,
     networkId: 'kovan',
     startBlock: 21406294,
     version: 3
   },
 ]
 
+export let poolMetaByAddress = new Map<string, PoolMeta>()
```

Comment:
If I understand this correctly, this can replace the separate `poolMetaById`, `poolMetaByShelf`, .. lookups, right? Maybe it would be good to actually replace those with this single map then, to reduce memory requirements?

Reply:
yep, i think i'll do this in a separate pr

Reply:
done